snuba/datasets/entities/outcomes.py

```diff
@@ -66,7 +63,8 @@ def __init__(self) -> None:
             writable_storage=writable_storage,
             validators=[EntityRequiredColumnValidator({"org_id"})],
             required_time_column="timestamp",
-            validate_data_model=ColumnValidationMode.WARN,
```

Comment:
Why don't you just use the `DO_NOTHING` value here?

Reply:
The `Entity` base class defaults to `DO_NOTHING`. Setting it to `WARN` or `ERROR` here is to override it on an entity by entity basis.

https://github.com/getsentry/snuba/blob/655a05079fee0828f72e2f05fa2cf8946f11826b/snuba/datasets/entity.py#L37